geomstats/geometry/general_linear_group.py

```diff
@@ -34,9 +39,16 @@ def get_identity(self, point_type=None):
                 ' implemented for a point_type that is not \'matrix\'.')
     identity = property(get_identity)
 
-    def belongs(self, mat):
-        """
-        Check if mat belongs to GL(n).
+    def belongs(self, mat):  # TODO Should this stay generic? (point)
```

Comment:
Good question, I could not decide on this. A TODO makes sense here

Reply:
What's the best way to annotate these things? I don't want them to get exposed to the users unnecessarily, but I also want contributors to see them. Hmm.